docs/conf.py

```diff
@@ -1,109 +1,34 @@
-templates_path = ['_templates']
-
-# The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-source_suffix = ['.rst']
-
-# The master toctree document.
-master_doc = 'index'
+extensions = [
+    "sphinx_copybutton",
+    "sphinx_design",
+    "sphinx.ext.mathjax",
+    "sphinxext.rediraffe",
+    "myst_parser",
+]
+templates_path = []
+source_suffix = [".rst", ".md"]
+root_doc = "index"
 
 # General information about the project.
-project = 'Team Compass'
-copyright = '2022, JupyterHub Team and Binder Team'
-author = 'JupyterHub Team and Binder Team'
-
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-#
-# The short X.Y version.
-version = '1.0'
-# The full version, including alpha/beta/rc tags.
-release = '1.0'
-
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = None
-
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
-
-# If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = False
-
-
-def setup(app):
-    app.add_css_file("custom.css")
+project = "Team Compass"
+copyright = "2022, JupyterHub Team and Binder Team"
+author = "JupyterHub Team and Binder Team"
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
 
 # -- Options for HTML output ----------------------------------------------
-
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-html_theme = 'pydata_sphinx_theme'
-
-# Logo
-html_logo = '_static/logo.png'
-html_favicon = '_static/favicon.png'
-
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
-#
-# html_theme_options = {}
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_theme = "pydata_sphinx_theme"
+html_logo = "_static/logo.png"
+html_favicon = "_static/favicon.png"
+html_static_path = ["_static"]
 html_theme_options = {
     "use_edit_page_button": True,
-    "google_analytics_id": "UA-101904940-3",
+    "analytics": {
+        "google_analytics_id": "UA-101904940-3",
+    },
```

Comment:
I forgot! Doing `make html` this showed up.

```
preparing documents... WARNING: unsupported theme option 'analytics' given
```

Looking at https://github.com/pydata/pydata-sphinx-theme/pull/650/files it seems like it should remain as it were, where `google_analytics_id` is written out directly under html_theme_options.

```suggestion
    "google_analytics_id": "UA-101904940-3",
```

Reply:
Or hmmm... This indicates we should do it like you suggested.
https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/analytics.html

Reply:
I had an outdated version of the theme installed.